easybuild/easyconfigs/l/lagrangian-filtering/lagrangian-filtering-0.8.3-foss-2022a.eb

```diff
@@ -0,0 +1,83 @@
+# This installation unfortunately requires the GitHub dance, as the tarball
+# is simply not installing. For that reason there is no checksum! 
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'lagrangian-filtering'
+version = '0.8.3'
+
+homepage = 'https://github.com/angus-g/lagrangian-filtering'
+description = """Temporal filtering of data in a Lagrangian frame of reference.""" 
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('dask', '2022.10.2'),
```

Comment:
There is already `scikit-image-0.19.3-foss-2022a.eb` depending on dask v2022.10.0 in this toolchain. Please downgrade this dependency to that version.